scripts/scr_ui_manage/scr_ui_manage.gml

```diff
@@ -1,46 +1,69 @@
 function scr_ui_manage() {
+	
+	var romanNumerals=[
+			"I",
+			"II",
+			"III", 
+			"IV", 
+			"V", 
+			"VI", 
+			"VII", 
+			"VIII", 
+			"IX", 
+			"X",
+		];
+		
+	var normal_hp=true;
+	
+	// Declare non marine roles here
+	var non_marine_roles=[
```

Comment:
we should really switch to string ids (e.g. sister_of_battle). But that's for the future.